nncf/torch/quantization/quantize_model.py

```diff
@@ -26,45 +26,41 @@
 from nncf.quantization.advanced_parameters import apply_advanced_parameters_to_config
 from nncf.scopes import IgnoredScope
 from nncf.scopes import convert_ignored_scope_to_list
-from nncf.torch.dynamic_graph.context import no_nncf_trace
-from nncf.torch.dynamic_graph.io_handling import replicate_same_tensors
-from nncf.torch.dynamic_graph.io_handling import wrap_nncf_model_inputs_with_objwalk
-from nncf.torch.dynamic_graph.io_handling import wrap_nncf_model_outputs_with_objwalk
 from nncf.torch.initialization import PTInitializingDataLoader
 from nncf.torch.model_creation import create_compressed_model
-from nncf.torch.nested_objects_traversal import objwalk
 from nncf.torch.nncf_module_replacement import replace_modules_by_nncf_modules
 from nncf.torch.quantization.weights_compression import insert_pre_compression_operations
-from nncf.torch.utils import get_model_device
-from nncf.torch.utils import is_tensor
 
 DEFAULT_RANGE_TYPE = "mean_min_max"
 
 
 # TODO(alexsu52): It is a workaround and should be removed.
-class CalibrationDataLoader(PTInitializingDataLoader):
+class CalibrationDataLoaderAdapter(PTInitializingDataLoader):
```

Comment:
What is the reason for these changes?

Reply:
This class doesn't seem to be a data loader that is functional on its own (e.g. instantiated with arbitrary data sources), its only purpose is to serve as an adapter from the `nncf.Dataset` to a `PTInitializingDataLoader`, so I renamed it accordingly.

Reply:
This changes are not aligned with the PR description. I suggest to do it in a separate PR.

Reply:
This is a related improvement, so you can't dismiss it on these grounds alone. But this name is not exposed, so there's no good reason for me to push it.